src/main/resources/io/jenkins/plugins/designlibrary/Layouts/twoColumn.jelly

```diff
@@ -0,0 +1,24 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout">
+  <l:layout title="Heading">
+    <l:side-panel>
+      <l:app-bar title="Heading" />
```

Comment:
Should the side panel also have an app-bar? Or is this a left-over? All other Jenkins views only use tasks.

Reply:
they _can_ have one if they have a heading in the side-panel. (uncommon)